news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -12,115 +12,118 @@ tags:
+Although it is very hard to estimate the size of this scope, we try to size it in order for the development phase to last 4 months[1].
+
+Once this scope has been is completed – i.e all issues have reached the "Done" column of the Kanban – the project reaches the [Feature Freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)) stage.
 
-When this scope is completed - i.e all github issues have reached the "Done" column of the Kanban - we trigger the [Feature Freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)).
+During this phase, no new items can be added to the version's scope, unless they are bugs related to code changes performed during the development of this version – called _regressions_. However, this is a _Feature Freeze_, not a _Code Freeze_, so some older bugs may be added to the scope if it is considered opportune to fix them quickly before the release is out (e.g. security fixes).
 
-We however accept that some issues remain in the Kanban, outside of "Done" column, only if they are bugs, no features. This is a Feature Freeze, not a Bug Freeze.
+## Feature Freeze
 
-# Feature Freeze
+Feature freeze means that all features of this version have been done and no new ones may be accepted in its scope. The project enters a phase of _stabilization_ whose aim is to identify and fix all bugs related to changes in the code during the development of this version (called _regressions_) before it's released.
 
-Feature freeze means that we freeze the development of the version and enter a phase of stabilization, which aim is naturally the release some time later.
+Once this phase is started, Core maintainers create a git branch from `develop` branch which will carry the work to be done until the release (for 1.7.7.0, the branch name is `1.7.7.x`). From this moment on, only bug fixes can be merged into this branch. Incidentally, this is also the branch where all future patch versions from this minor version will be developed on (hence the `.x` at the end).
 
-We create a git branch from `develop` branch which will carry the work to be done until the release (for 1.7.7.0, the branch name is `1.7.7.x`). From this moment, only bug fixes can be merged into this branch.
+Also when feature freeze is reached for `1.7.7.x` branch, it means than the next minor release (1.7.8) development phase starts on `develop` branch ! A minor version development actually starts (albeit slowly) precisely when the previous version enters feature freeze.
 
-Maintainers create an Alpha Build from the `1.7.7.x` branch codebase and deliver it to QA team which starts a huge test campaign. The goal of this campaign is to find and register all regressions[1] of this build.
+Maintainers create an Alpha Build from the `1.7.7.x` branch codebase and deliver it to QA team which starts a huge test campaign. The goal of this campaign is to find and register all regressions[2] of this build.
```

Comment:
The term _regressions_ has been introduced earlier.

```suggestion
Maintainers create an Alpha Build from the `1.7.7.x` branch codebase and deliver it to QA team which starts a huge test campaign. The goal of this campaign is to find and register all regressions of this build.
```

I'd suggest agains mentioning an alpha build, people will think that we do it just because we wrote about it. We've been talking about nightly builds for a [long time now](https://www.google.com/search?biw=2560&bih=1306&ei=g3p4XqjTOPvRgwfskqfQBA&q=site%3Abuild.prestashop.com+nightly+builds&oq=site%3Abuild.prestashop.com+nightly+builds&gs_l=psy-ab.3...7273.7905..8270...0.0..0.43.340.9......0....1..gws-wiz.7HF0vrE8Skg&ved=0ahUKEwjooL7bnbDoAhX76OAKHWzJCUo4ChDh1QMICw&uact=5). 

```suggestion
The QA team picks up the latest [nightly build](https://nightly.prestashop.com/) and starts a huge test campaign. The goal of this campaign is to find and register all regressions of this build.
```